src/main/java/Duke.java

```diff
@@ -1,10 +1,82 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static Task[] tasks = new Task[100];
+
+    /** Prints a farewell message when the user exits the program. */
+    public static void commandBye() {
```

Comment:
Maybe this method can be made private as it is not being used by any other class